addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -622,6 +626,67 @@ options.registry.Theme = options.Class.extend({
 
         await this._reloadBundles();
     },
+    /**
+     * @see this.selectClass for parameters
+     */
+    async openCustomCodeDialog(previewMode, widgetValue, params) {
+        let websiteId;
+
+        let fieldName, contentText;
+        if (widgetValue === 'head') {
+            fieldName = 'custom_code_head';
+            contentText = _t('Enter code that will be added into the <head> of every page of your site.');
+        } else {
+            fieldName = 'custom_code_footer';
+            contentText = _t('Enter code that will be added before the </body> of every page of your site.');
+        }
+
+        this.trigger_up('context_get', {
```

Comment:
You can move the `websiteId` declaration just before the call to `trigger_up` here, it would be a bit clearer to me.